html/includes/dev-overview-data.inc.php

```diff
@@ -16,7 +16,13 @@
 echo '</div>
       <table class="table table-hover table-condensed table-striped">';
 
-$uptime = $device['uptime'];
+$uptime = formatUptime($device['uptime']);
```

Comment:
Minor, but shouldn't this be in an else?

Reply:
Doesn't need to be. We set it to what it was before and overwrite it if the device is down using another value.